Standards/CCSMB-10.md

```diff
@@ -0,0 +1,14 @@
+# *CCSMB 10:* Specifying where programs and libraries are installed and where user data is stored
+
+*Author: @Rainb0wSkeppy*  
+*Version: vX.Y.Z*  
+*Last updated: YYYY-MM-DD*  
+
+## `settings` API settings
+Compliant installers MUST install programs into the directory specified through the `path.programs` setting and MUST install libraries into the directory specified through the `path.libraries` setting.
+The installers SHOULD NOT run if the settings aren't set.  
+If the program or library is multiple files, the files MUST be installed in a subdirectory.
+
+Compliant programs MUST store user data into the directory specified through the `path.data` setting.
+The programs SHOULD NOT run if the setting isn't set.
```

Comment:
I would want my programs to run even if I haven't set the setting. It will be very confusing for users not aware of the CCSMB, who just play around with ComputerCraft.

Reply:
Maybe ask where to put user data for programs and then set the setting